packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -1575,8 +1575,14 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   'timeline.error.unable-to-load-revision': 'Unable to load revision',
   /** Label for when the timeline item is the latest in the history */
   'timeline.latest': 'Latest',
-  /** Label for latest version for timeline menu dropdown */
-  'timeline.latest-version': 'Latest version',
+
+  /** Label for latest revision for timeline menu dropdown */
+  'timeline.latest-revision': 'Latest revision',
+  /**
+   * Label for latest revision for timeline menu dropdown
+   * @deprecated use `timeline.latest-revision` instead
+   * */
+  'timeline.latest-version': 'Latest revision',
```

Comment:
defer to @rexxars on deprecating i18n keys. Not sure we have a process

Reply:
There's no clear process, but we can make this an opportunity to establish one?

This was a fair amount of OCD-induced change 😅, but I still feel the change it's justified by the likelyhood of existing language bundles already having translated "version" instead of "revision" (for example, the [norwegian bundle](https://github.com/sanity-io/locales/blob/main/locales/nb-NO/src/studio.ts#L1542) is currently using "Version" here, when it should be "Revisjon"), and this change will give downstream locale implementors a chance to notice the change.

I've pushed some added clarity in the comments in 4864109d596d9d9757b8e5a2caacc212a78d2837 based on input from @rexxars.

Reply:
understood, I am okay with the change just wanted to call it out